Sources/SCNPath/RKPathEntity.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  RKPathEntity.swift
+//  SCNPath+Example
+//
+//  Created by Grant Jarvis on 3/13/21.
+//  Copyright © 2021 Max Cobb. All rights reserved.
+//
+
+import SceneKit
+import Combine
+import RealityKit
+
+public class RealityKitPathEntity : SCNPathNode {
+    
+    ///The SceneKit scene that this node belongs to.
+    public weak var scene : SCNScene?
+    
+    ///A realityKit entity which represents the generated path.
+    public var realityKitPath = Entity()
+    
+    ///The number of times we have generated a .usdz file
+    ///This is used for changing the file path each time.
+    private var entityCount = 0
+    
+    private var circleEntity : Entity?
+    
+    private var rectangleEntity : Entity?
+    
+    private var pathEntitiesInScene = [Entity]()
+    
+    override init(
+        path: [SCNVector3], width: Float = 0.5,
+        curvePoints: Float = 8, materials: [SCNMaterial] = []
+    ) {
+        super.init(path: path,
+                   width: width,
+                   curvePoints: curvePoints,
+                   materials: materials)
+        do {
+            let shapesScene = try ShapesProj.loadShapesScene()
+            loadShapes(scene: shapesScene)
+        } catch {
+            print(error.localizedDescription)
+            return
+        }
+        //Remove synchronization to save memory.
+        self.realityKitPath.visit(using: {$0.synchronization = nil})
```

Comment:
That’s good to know!
This code with `.visit()` was taken directly from Apple’s Swift Strike Table Top app, inside of ViewController.swift, lines 247 - 251 and lines 360 - 368. "Disabling the synchronization of entities to reduce the memory consumption (since this is a single-player experience)."